src/language-server/logger.ts

```diff
@@ -19,9 +19,20 @@ import Settings from './settings';
 
 export class Logger {
   private logFile: WriteStream|undefined = undefined;
+  private readonly logFileFromFlag: WriteStream|undefined;
```

Comment:
This name feels weird, as it is not a flag but actually a stream. Why not `logFileStream`?

Reply:
It's not the flag itself, it's the file that the user asked us to log to based on a command line flag.